src/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,69 @@
+
+## PROS AND CONS {#pros-and-cons}
+
+| Pros                                                                                                                                                      | Cons                                                                                                                                                  |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Scalable more than other popular mechanisms such POS and POW, as it's based on a limited number of block signers                                          | PoA networks typically have a relatively small number of validating nodes. This makes a PoA network more centralized.                                 |
+| PoA blockchains are incredibly cheap to run and maintain                                                                                                  | becoming an authorized signer is typically out of reach for an ordinary person, because the blockchain requires entities with established reputation. |
+| The transactions are confirmed very quick as it could reach less than 1 second because only limited number of signers are required to validate new blocks | malicious signers could reorg,double spend,censor transactions in the network, those attacks are mitigated but still possible                         |
+
+## Further Reading {#further-reading}
+
+- [EIP-225](https://eips.ethereum.org/EIPS/eip-225) _Clique standard_
+- [proof of authority study](https://github.com/cryptoeconomics-study/website/blob/master/docs/sync/2.4-lecture.md) _Cryptoeconomics_
+- [What is Proof of Authority](https://forum.openzeppelin.com/t/proof-of-authority/3577) _OpenZeppelin_
+- [Proof of Authority Explained](https://academy.binance.com/en/articles/proof-of-authority-explained) _binance_
+- [PoA in blockchain](https://medium.com/techskill-brew/proof-of-authority-or-poa-in-blockchain-part-11-blockchain-series-be15b3321cba)
+- [Clique explained](https://medium.com/@Destiner/clique-cross-client-proof-of-authority-algorithm-for-ethereum-8b2a135201d)
+- [deprecated PoA, Aura specification](https://openethereum.github.io/Chain-specification)
+- [IBFT 2.0, another PoA implementation](https://besu.hyperledger.org/stable/private-networks/concepts/poa)
+
+### Videos {#videos}
+
+- [Visual explanation of Proof of Authority](https://youtu.be/Mj10HSEM5_8)
```

Comment:
```suggestion
## More of a visual learner? {#visual-learner}

Watch a visual explanation of proof-of-authority:

<YouTube id="Mj10HSEM5_8" />
```

Reply:
This is a pattern we've established on a lot of other pages (this header, followed by embedding the YouTube video)... we typically will place this [above "Further reading" as well](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/pos-vs-pow/#visual-learner).